src/privateStore.js

```diff
@@ -65,30 +72,52 @@ class PrivateStore {
    * @param     {String}    hash                        The hash of the private store OrbitDB
    */
   async _sync (hash) {
-    // if hash === null; create a new orbitdb instance.
-    const orbitdb = new OrbitDB(this.ipfs)
-    //TODO - sync the OrbitDB key-value store
+    if (!this.db) {
+      const orbitdb = new OrbitDB(this.ipfs)
+      this.db = await orbitdb.keyvalue('3box.datastore', {
```

Comment:
Good point, right now it doesn't matter since we only sync using hashes anyway. However if we at some point in the future enable syncing with pubsub this could screw things up. We should probably put some unique name for each user here.